examples/Org.OData.Aggregation.V1.SalesModel-sample.xml

```diff
@@ -81,13 +93,20 @@
           <PropertyRef Name="ID" />
         </Key>
         <Property Name="ID" Type="Edm.String" Nullable="false" />
-        <Property Name="Amount" Type="Edm.Decimal" Scale="2"/>
+        <Property Name="Amount" Type="Edm.Decimal" Scale="2">
+          <Annotation Term="Aggregation.ContextDefiningProperties">
+            <Collection>
+              <PropertyPath>Currency/Code</PropertyPath>
+            </Collection>
+          </Annotation>
+        </Property>
+        <NavigationProperty Name="Currency" Type="SalesModel.Currency" Nullable="false" />
         <NavigationProperty Name="SalesOrganization" Type="SalesModel.SalesOrganization" Partner="Sales" />
         <NavigationProperty Name="Product" Type="SalesModel.Product" Partner="Sales" />
         <NavigationProperty Name="Customer" Type="SalesModel.Customer" Partner="Sales" />
         <NavigationProperty Name="Time" Type="SalesModel.Time" />
       </EntityType>
-      <EntityContainer Name="Container">
+      <EntityContainer Name="SalesModel">
```

Comment:
Same name as the containing schema, which might cause confusion, for example in 
~~~xml
<Annotation Target="SalesModel.SalesModel/Sales"
~~~
Why not leave it as "Container" or "defaultContainer"?

Reply:
Also now the annotation target in line 131 is broken, still says `Target="SalesModel.Container/Sales"`

Reply:
> Why not leave it as "Container" or "defaultContainer"?

I wanted to follow section 6.4 in OData-Aggr.